common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -1238,7 +1238,10 @@ def submit_problem(self, data, override_time=False):
             # without a stack trace
             else:
                 # Translators: {msg} will be replaced with a problem's error message.
-                msg = _(u"Error: {msg}").format(msg=inst.message)
+                if type(inst) == StudentInputError:
```

Comment:
I think we should just remove Error.  I do not believe it adds value.  

As for reviewing the error messages this could be valuable, but might take some time.  We would need to review every capa response type.

@cahrens @srpearce Are you ok with me removing the Error completely for now and ensuring the messages start appropriately?

Or would you rather I change it to Warning?